src/QMCWaveFunctions/MolecularOrbitals/AtomicBasisBuilder.h

```diff
@@ -239,7 +239,7 @@ typename AtomicBasisBuilder<RFB>::COT* AtomicBasisBuilder<RFB>::createAOSet(xmlN
     if (cname1 == "basisGroup")
     {
       radGroup.push_back(cur1);
-      int l = atoi((const char*)(xmlGetProp(cur1, (const xmlChar*)"l")));
+      const int l = std::stoi(XMLAttrString{cur1, "l"});
```

Comment:
I want to get to using the c++ coversion functions but std::stoi will throw an exceptions if it can't parse the input. I don't think this is in a try catch scope.

Reply:
At the moment, I think it is no worse than the C functions
When I was changing atoi to stoi and atof to stod, I did ask myself what happens if the conversion fails. I think we do need to catch the exception which is good for the code. I thought about using istringstream and check failbit but the code gets lengthy.
Could you explain a bit more what you mean by  "I don't think this is in a try catch scope"?